### ISSUE TITLE
short cut keys for the inc and dec width and height

### ISSUE THREAD

**Issue by elieserdejesus** [elieserdejesus/JamTaba#936]
Proposed by Andy:

>  Would be great having couple or short cut keys for the inc and dec width and height, like CTRL + keys or ctrl and mouse wheel or something. Is a bit fiddly goign back to top file menu to adjust size.

**Comment by Hitrate** [comment]
Good idea. Or auto-sizing so we would not have to adjust the width manually depending on how many people are in the room, leaving and coming.

**Comment by Hitrate** [comment]
*And

**Comment by elieserdejesus** [comment]
Andy suggested in another issue:
```
Could be a like clickable boxes next to the other resize buttons.
H + -
W + -
```
